src/main/kotlin/org/opensearch/replication/seqno/RemoteClusterStats.kt

```diff
@@ -0,0 +1,92 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ *
+ * Modifications Copyright OpenSearch Contributors. See
+ * GitHub history for details.
+ */
+
+package org.opensearch.replication.seqno
+
+import org.opensearch.common.component.AbstractLifecycleComponent
+import org.opensearch.common.inject.Singleton
+import org.opensearch.common.io.stream.StreamInput
+import org.opensearch.common.io.stream.StreamOutput
+import org.opensearch.common.xcontent.ToXContent
+import org.opensearch.common.xcontent.ToXContentObject
+import org.opensearch.common.xcontent.XContentBuilder
+
+class RemoteIndexStat: ToXContentObject {
+    var ops :Long = 0
+    var tlogSize :Long = 0
+    var opsLucene :Long = 0
+    var opsTlog :Long = 0
+    var latencyLucene :Long = 0
+    var latencyTlog :Long = 0
+    var bytesRead :Long = 0
+
+    constructor()
+
+    constructor(inp: StreamInput)  {
+        ops = inp.readLong()
+        tlogSize = inp.readLong()
+        opsLucene = inp.readLong()
+        opsTlog = inp.readLong()
+        latencyLucene = inp.readLong()
+        latencyTlog = inp.readLong()
+        bytesRead = inp.readLong()
+    }
+
+    fun writeTo(out: StreamOutput) {
+        out.writeLong(ops)
+        out.writeLong(tlogSize)
+        out.writeLong(opsLucene)
+        out.writeLong(opsTlog)
+        out.writeLong(latencyLucene)
+        out.writeLong(latencyTlog)
+        out.writeLong(bytesRead)
+    }
+
+    fun add(stat :RemoteIndexStat): RemoteIndexStat {
+        var newStat = this
+        newStat.ops += stat.ops
+        newStat.tlogSize += stat.tlogSize
+        newStat.opsLucene += stat.opsLucene
+        newStat.opsTlog += stat.opsTlog
+        newStat.latencyLucene += stat.latencyLucene
+        newStat.latencyTlog += stat.latencyTlog
+        newStat.bytesRead += stat.bytesRead
+
+        return newStat
+    }
```

Comment:
I was refering to add method as the only way to add stats. This would have meant even in a place where I need to add bytesRead, I'd have had ton construct Stats object.

However latest rev is fine as we are individually adding the elements.

Reply:
Yes. This is used only while collecting and adding stats from all the nodes .